cdap-data-fabric-tests/src/test/java/co/cask/cdap/data2/dataset2/lib/table/hbase/HBaseTableTest.java

```diff
@@ -89,18 +91,24 @@
   @BeforeClass
   public static void beforeClass() throws Exception {
     cConf = CConfiguration.create();
-    hBaseTableUtil = new HBaseTableUtilFactory(cConf).get();
+    hBaseTableUtil = new HBaseTableUtilFactory(cConf, new SimpleNamespaceQueryAdmin()).get();
     // TODO: CDAP-1634 - Explore a way to not have every HBase test class do this.
-    hBaseTableUtil.createNamespaceIfNotExists(TEST_HBASE.getHBaseAdmin(), NAMESPACE1);
-    hBaseTableUtil.createNamespaceIfNotExists(TEST_HBASE.getHBaseAdmin(), NAMESPACE2);
+    hBaseTableUtil.createNamespaceIfNotExists(TEST_HBASE.getHBaseAdmin(),
```

Comment:
The idea is that all users of HBaseTableUtil should always convert to HBase entities before calling the methods. For example ``TableId`` and ``namespace``. Basically, HBaseTableUtil only understands ``HBase`` entities. Now the conversion from CDAP to HBase entities are also provided by the HBaseTableUtil which basically uses NamespaceQueryAdmin. Now the latter part (where those conversions are done or what is used for conversion) might change, especially after CDAP-6534 is addressed